server.py

```diff
@@ -1,19 +1,28 @@
 import socket
+from network_terminal_output import NetworkTerminalOutput
+from gamebook.terminal_output import TerminalOutput
 
 
-HOST = '127.0.0.1'
+HOST = '0.0.0.0'
 PORT = 6543
 
+server_output = TerminalOutput()
+
 with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as socket:
     socket.bind((HOST, PORT))
-    socket.listen()
-
-    client, address = socket.accept()
-    with client:
-        print(f"connection made from {address}")
-        while True:
-            data = client.recv(1024)
-            if not data:
-                break
-            client.send(data)
-
+    socket.listen(5)
```

Comment:
why 5?

Reply:
forgot to remove it after i finished experimenting